content/200-orm/200-prisma-client/000-setup-and-configuration/006-ts-only-client.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'TS-only Prisma Client'  
+metaTitle: 'TypeScript-only Prisma Client'  
+metaDescription: 'Learn about the TypeScript-only Prisma Client.'  
+tocDepth: 3  
+sidebar_class_name: preview-badge
+community_section: true
+---
+
+The **TypeScript-only Prisma Client** improves compatibility for Prisma ORM across various Node.js environments, including those using ECMAScript modules (ESM), and is [currently in Preview](/orm/more/releases#preview). 
+
+This approach lets you bundle your application code as needed while keeping the Prisma type definitions generated outside of `node_modules` and in your specified folder. With everything explicitly defined, you avoid hidden magic and gain greater control over your project's configuration.
+
+## Getting started
+
+Follow these steps to generate and integrate the TS-only Prisma Client into your project.
+
+### 1. Change the provider in the generator block
+
+First, update the [`generator` block](/orm/prisma-schema/overview/generators) in your `schema.prisma` file. Ensure that both the `provider` and the `output` path are correctly configured. This step tells Prisma to use the TypeScript-only client and determines where the generated code will reside.
+
+```prisma file=schema.prisma
+generator client {
+  provider = "prisma-client-ts"
+  output   = "./generated/prisma-client-ts"
```

Comment:
`../generated` or `../src/generated`.

We should not encourage users to generate the client in the `prisma` directory, they should be kept separate.

Also, why `prisma-client-ts` and not just `prisma`? `generated/prisma-client-ts/client` is a tautology, `generated/prisma/client` is much nicer.